client/core/core.go

```diff
@@ -3405,6 +3411,11 @@ func (c *Core) verifyAccount(acct *db.AccountInfo) bool {
 	dc, err := c.connectDEX(acct)
 	if err != nil {
 		c.log.Errorf("error connecting to DEX %s: %v", acct.Host, err)
+		// update failed connection, so we can give feedback.
+		c.connMtx.Lock()
+		c.conns[host] = dc
+		c.connMtx.Unlock()
```

Comment:
In the case of an error, `dc` will be `nil`. Might as well just `c.conns[host] = nil` here so there's no question. Though if we're going to use a nil-valued entry like this, the intention needs to be very clearly documented somewhere.

Reply:
dc will not be nil in this case, because now I am returning dc's value if it is disconnect, as well: https://github.com/decred/dcrdex/pull/989/files#diff-13a4a6b12c1ae844ef90151638f194427cfdc545fceb58d52c0f6f33b81d859bR4072

Reply:
My bad. I meant to delete this comment before submitting my review.

Reply:
My bad. I meant to delete this comment before submitting my review.